src/main/java/com/google/sps/servlets/MentorSignupServlet.java

```diff
@@ -32,9 +35,46 @@
 @WebServlet("/mentor-signup")
 public class MentorSignupServlet extends HttpServlet {
 
+  // Set up variables needed to connect to MySQL database.
+  String url = Utility.SQL_URL;
+  String user = Utility.SQL_USER;
+  String password = Utility.SQL_PASSWORD;
+
   /**
-   * Receives information about a new mentor and stores it in the database.
+   * Gets mentor experience tags from database and returns as JSON.
    */
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Set up list to store subject tags.
+    List<SubjectTag> subjectTags = new ArrayList<SubjectTag>();
+    
+    // Set up query to retrieve all subject tags.
+    String query = "SELECT * FROM SubjectTag";
+
+    try {
+      // Establish connection to MySQL database.
+      Connection connection = DriverManager.getConnection(url, user, password);
+
+      // Create the MySQL prepared statement, execute it, and store the result.
+      PreparedStatement preparedStatement = connection.prepareStatement(query);
+      ResultSet queryResult = preparedStatement.executeQuery();
+
+      // Store queryResult in map and close connection to database.
+      while (queryResult.next()) {
+        subjectTags.add(new SubjectTag(
+            queryResult.getInt(1), queryResult.getString(2), queryResult.getString(3)));
+      }
```

Comment:
I think it is more common standard to include everything inside the try block, instead of the parenthesis. Examples:
- https://google.github.io/styleguide/javaguide.html#s4.1.2-blocks-k-r-style
- https://google.github.io/styleguide/javaguide.html#s6.2-caught-exceptions
- https://www.w3schools.com/java/java_try_catch.asp